crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -65,9 +66,27 @@ where
         }
     }
 
-    pub fn new_formatting(format: S, children: Vec<DomNode<S>>) -> Self {
+    pub fn new_formatting_from_tag(
+        format: S,
+        children: Vec<DomNode<S>>,
+    ) -> Self {
         Self {
             name: format.clone(),
+            kind: ContainerNodeKind::Formatting(
+                InlineFormatType::try_from(format).unwrap(),
```

Comment:
I suggest instead of unwrapping here we should return an Option, so the caller can decide whether unwrap is the right way to handle it, because the caller knows whether the tag is known.

Reply:
Do you mean returning `Option<Self>` here depending on whether the tag is valid or not?